### ISSUE TITLE
Normalisation of the confusion matrix

### ISSUE THREAD

**Issue by AndresAlgaba** [fairlearn/fairlearn#1173]
First of all, thank you to all the developers of fairlearn for this amazing package :)!

#### Describe the bug
I was looking into the disparities between error rates with the ``true/false_positive/negative_rate`` functions from: https://github.com/fairlearn/fairlearn/blob/main/fairlearn/metrics/_extra_metrics.py.

I found some unexpected results when I compared the rates with my own calculations and found that the source of confusion arised from the normalisation method. The current setting in the ``confusion_matrix`` from ``sklearn`` is ``normalize = "true"``, after which the rates are returned. However, this means that there is only a normalization over the rows. Intuitively, when I compute the rates myself, I normalise over the entire population, which corresponds to setting ``normalize = "all"``.

If you agree that this is an issue, I would be willing to implement a PR!

#### Steps/Code to Reproduce
Below I provide an example of the two different normalisation methods, and my own custom computation which boils down to just normal division:

```python
import numpy as np
from sklearn.metrics import confusion_matrix

rng = np.random.default_rng(42)

target = rng.integers(0, 2, 100000)
preds = rng.integers(0, 2, 100000)

tn, fp, fn, tp = confusion_matrix(target, preds).ravel()
tn_true, fp_true, fn_true, tp_true = confusion_matrix(target, preds, normalize="true").ravel()
tn_all, fp_all, fn_all, tp_all = confusion_matrix(target, preds, normalize="all").ravel()

print(((tp) / (tn+fp+fn+tp)), tp_true, tp_all)

0.24939 0.4978937491265547 0.24939
```

#### Expected Results
The expected result is that the normalisation corresponds to dividing the number of, for example, true positives by the entire population, which is currently not the case.

#### Actual Results
Currently, there is only a normalisation over the rows by setting ``normalize = "true"``.

#### Versions
System:
    python: 3.9.15 (main, Nov 15 2022, 21:37:24)  [GCC 10.2.1 20210110]
executable: /usr/local/bin/python
   machine: Linux-5.15.49-linuxkit-x86_64-with-glibc2.31

Python dependencies:
    Cython: None
matplotlib: 3.6.2
     numpy: 1.23.5
    pandas: 1.5.2
       pip: 22.0.4
     scipy: 1.9.3
setuptools: 58.1.0
   sklearn: 1.1.0
    tempeh: None

**Comment by hildeweerts** [comment]
Hi, @AndresAlgaba - super glad to hear you like Fairlearn!

Fairlearn uses the "standard" definition of false/true positive/negative rates. For example, the false positive rate is equal to $fpr = fp / (fp +tn)$, i.e., the number of false positive divided by the number of ground-truth negatives. I always forget which one is which myself, so I often refer back to this Wikipedia page: https://en.wikipedia.org/wiki/Sensitivity_and_specificity 

I hope this helps - please let me know if this answers your question.

**Comment by AndresAlgaba** [comment]
Hi @hildeweerts, thanks for your quick reply :D It was a silly mistake on my end to divide by the population, indeed! Thanks for the clarification!
